diff --git a/other-licenses/7zstub/firefox/resource.rc b/other-licenses/7zstub/firefox/resource.rc
--- a/other-licenses/7zstub/firefox/resource.rc
+++ b/other-licenses/7zstub/firefox/resource.rc
@@ -1,7 +1,7 @@
 #include <windows.h>
 #include "../src/CPP/7zip/Bundles/SFXSetup/resource.h"
 #include "../src/C/7zVersion.rc"
 
 LANGUAGE 9, 1
 1 VERSIONINFO
   FILEVERSION MY_VER
@@ -11,14 +11,14 @@
   FILEOS MY_VOS_NT_WINDOWS32
   FILETYPE MY_VFT_APP
   FILESUBTYPE 0x0L
 BEGIN
   BLOCK "StringFileInfo"
   BEGIN
       BLOCK "040904b0"
       BEGIN
-          VALUE "CompanyName", "Mozilla"
-          VALUE "FileDescription", "Firefox"
+          VALUE "CompanyName", "ASYS Technology"
+          VALUE "FileDescription", "Vantage"
           VALUE "FileVersion", MY_VERSION
           VALUE "InternalName", "7zSD.sfx"
-          VALUE "LegalCopyright", "Mozilla"
+          VALUE "LegalCopyright", "ASYS Technology"
           VALUE "OriginalFilename", "7zSD.sfx"
-          VALUE "ProductName", "Firefox"
+          VALUE "ProductName", "Vantage"
           VALUE "ProductVersion", MY_VERSION
       END
   END
   BLOCK "VarFileInfo"
