diff -Naur a/other-licenses/7zstub/firefox/resource.rc b/other-licenses/7zstub/firefox/resource.rc
--- a/other-licenses/7zstub/firefox/resource.rc	2026-05-10 15:45:03.988832389 +0800
+++ b/other-licenses/7zstub/firefox/resource.rc	2026-05-10 15:45:03.992832658 +0800
@@ -16,13 +16,13 @@
   BEGIN
       BLOCK "040904b0"
       BEGIN
-          VALUE "CompanyName", "Mozilla"
-          VALUE "FileDescription", "Firefox"
+          VALUE "CompanyName", "ASYS Technology"
+          VALUE "FileDescription", "Vantage"
           VALUE "FileVersion", MY_VERSION
           VALUE "InternalName", "7zSD.sfx"
-          VALUE "LegalCopyright", "Mozilla"
+          VALUE "LegalCopyright", "ASYS Technology"
           VALUE "OriginalFilename", "7zSD.sfx"
-          VALUE "ProductName", "Firefox"
+          VALUE "ProductName", "Vantage"
           VALUE "ProductVersion", MY_VERSION
       END
   END
