--- a/browser/installer/windows/nsis/defines.nsi.in	2026-06-11 16:24:49.797171395 +0800
+++ b/browser/installer/windows/nsis/defines.nsi.in	2026-06-11 16:24:49.797171395 +0800
@@ -39,7 +39,7 @@
 !define BrandShortName        "@MOZ_APP_DISPLAYNAME@"
 !endif
 !ifndef BrandFullName
-!define BrandFullName         "${BrandFullNameInternal}"
+!define BrandFullName         "Vantage"
 !endif
 
 ; The C++ defines in `nsNativeAppSupportWin.h` are 1-based.  The Windows
