--- a/browser/installer/package-manifest.in
+++ b/browser/installer/package-manifest.in
@@ -264,6 +264,11 @@
 ; Remote Settings JSON dumps
 @RESPATH@/browser/defaults/settings
 
+; LibreWolf specific files.
+@RESPATH@/defaults/pref/local-settings.js
+@RESPATH@/distribution/policies.json
+@RESPATH@/librewolf.cfg
+
 # channel-prefs.js has been removed on macOS.
 #ifndef XP_MACOSX
 ; Warning: changing the path to channel-prefs.js can cause bugs (Bug 756325)
--- a/lw/moz.build
+++ b/lw/moz.build
@@ -0,0 +1,12 @@
+FINAL_TARGET_FILES += [
+  "librewolf.cfg",
+]
+
+FINAL_TARGET_FILES.distribution += [
+  "policies.json",
+]
+
+# local-settings does not yet end up being packaged.
+FINAL_TARGET_FILES.defaults.pref += [
+  "local-settings.js",
+]
--- a/moz.build
+++ b/moz.build
@@ -247,3 +247,5 @@
 SPHINX_TREES["jsloader"] = "docs/jsloader"
 
 include("build/templates.mozbuild")
+
+DIRS += ["lw"]
