--- a/browser/components/preferences/jar.mn
+++ browser/components/preferences/jar.mn
@@ -12,6 +12,7 @@
    content/browser/preferences/home.js
    content/browser/preferences/search.js
    content/browser/preferences/privacy.js
+   content/browser/preferences/librewolf.js
    content/browser/preferences/containers.js
    content/browser/preferences/sync.js
    content/browser/preferences/experimental.js

--- a/browser/components/preferences/preferences.js
+++ browser/components/preferences/preferences.js
@@ -8,6 +8,7 @@
 /* import-globals-from search.js */
 /* import-globals-from containers.js */
 /* import-globals-from privacy.js */
+/* import-globals-from librewolf.js */
 /* import-globals-from sync.js */
 /* import-globals-from experimental.js */
 /* import-globals-from moreFromMozilla.js */
@@ -397,6 +398,7 @@
   register_module("paneHome", gHomePane);
   register_module("paneSearch", gSearchPane);
   register_module("panePrivacy", gPrivacyPane);
+  register_module("paneLibrewolf", gLibrewolfPane);
   register_module("paneContainers", gContainersPane);
 
   if (ExperimentAPI.labsEnabled) {

--- a/browser/components/preferences/preferences.xhtml
+++ browser/components/preferences/preferences.xhtml
@@ -38,6 +38,7 @@
     href="chrome://browser/skin/preferences/containers.css"
   />
   <link rel="stylesheet" href="chrome://browser/skin/preferences/privacy.css" />
+  <link rel="stylesheet" href="chrome://browser/skin/preferences/librewolf.css" />
   <link rel="stylesheet" href="chrome://browser/content/preferences/widgets/setting-group.css" />
   <link rel="stylesheet" href="chrome://browser/content/preferences/widgets/setting-pane.css" />
   <link rel="stylesheet" href="chrome://browser/content/preferences/widgets/setting-control.css" />
@@ -147,10 +148,10 @@
         iconsrc="chrome://browser/skin/preferences/category-sync.svg"
         data-l10n-id="pane-sync-title3">
       </html:moz-page-nav-button>
-      <html:moz-page-nav-button id="category-ai-features"
-        view="paneAi"
-        iconsrc="chrome://global/skin/icons/highlights.svg"
-        data-l10n-id="pane-ai-controls-title">
+      <html:moz-page-nav-button id="category-librewolf"
+        view="paneLibrewolf"
+        iconsrc="chrome://browser/skin/preferences/category-librewolf.svg"
+        data-l10n-id="pane-librewolf-title">
       </html:moz-page-nav-button>
       <html:moz-page-nav-button id="category-experimental"
         hidden="true"
@@ -214,6 +215,7 @@
 #include home.inc.xhtml
 #include search.inc.xhtml
 #include privacy.inc.xhtml
+#include librewolf.inc.xhtml
 #include containers.inc.xhtml
 #include sync.inc.xhtml
 #include experimental.inc.xhtml

--- a/browser/themes/shared/jar.inc.mn
+++ browser/themes/shared/jar.inc.mn
@@ -152,6 +152,7 @@
   skin/classic/browser/preferences/android-menu.svg            (../shared/preferences/android-menu.svg)
   skin/classic/browser/preferences/category-general.svg        (../shared/preferences/category-general.svg)
   skin/classic/browser/preferences/category-privacy-security.svg    (../shared/preferences/category-privacy-security.svg)
+  skin/classic/browser/preferences/category-librewolf.svg    (../shared/preferences/category-librewolf.svg)
   skin/classic/browser/preferences/category-search.svg         (../shared/preferences/category-search.svg)
   skin/classic/browser/preferences/category-sync.svg           (../shared/preferences/category-sync.svg)
   skin/classic/browser/preferences/containers.css              (../shared/preferences/containers.css)
@@ -170,6 +171,7 @@
   skin/classic/browser/preferences/vpn-logo.svg                (../shared/preferences/vpn-logo.svg)
   skin/classic/browser/preferences/search.css                  (../shared/preferences/search.css)
   skin/classic/browser/preferences/siteDataSettings.css        (../shared/preferences/siteDataSettings.css)
+  skin/classic/browser/preferences/librewolf.css               (../shared/preferences/librewolf.css)
   skin/classic/browser/translations/panel.css                  (../shared/translations/panel.css)
   skin/classic/browser/fxa/fxa-spinner.svg                     (../shared/fxa/fxa-spinner.svg)
   skin/classic/browser/fxa/sync-illustration.svg               (../shared/fxa/sync-illustration.svg)
